caf-audit-monkey-auditlog/src/main/resources/xml/monkey-audit.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<!-- Copyright 2015-2017 Hewlett Packard Enterprise Development LP. Licensed 
+    under the Apache License, Version 2.0 (the "License"); you may not use this 
+    file except in compliance with the License. You may obtain a copy of the 
+    License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by 
+    applicable law or agreed to in writing, software distributed under the License 
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY 
+    KIND, either express or implied. See the License for the specific language 
+    governing permissions and limitations under the License. -->
+<AuditedApplication xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://cafaudit.github.io/audit-service/schema/AuditedApplication.xsd"
+    xsi:schemaLocation="https://cafaudit.github.io/audit-service/schema/AuditedApplication.xsd">
+    
+    <ApplicationId>AuditMonkey</ApplicationId>
+    <AuditEvents>
+        <AuditEvent>
+            <TypeId>viewDocument</TypeId>
+            <CategoryId>documentEvents</CategoryId>
```

Comment:
This and the two comments below regarding the simple nature of the Audit Events are fair.  I think this should / could be addressed if time allows or for the Audit 3.2.0 release.  Will see how today goes and how we are looking for Friday.  If there was not immediate dead line enhancing this would be ideal!